dask/distributed/worker.py

```diff
@@ -530,6 +530,11 @@ def close(self):
     def __del__(self):
         self.close()
 
+    def __repr__(self):
+        s = '<dask.distributed.Worker at %s address=%s, scheduler=%s>'
+        return s % (hex(id(self)), self.address.decode('utf-8')[6:],
+                    self.scheduler.decode('utf-8')[6:])
```

Comment:
We can probably leave off the hex(id(...)) bit and probably the `<dask.distributed` bit.  They don't provide much information and once we have the two addresses we've probably completely identified the worker, removing the need for the `id`.